src/main/java/ro/societateahermes/backendservice/service/serviceImplementation/EventServiceImplementation.java

```diff
@@ -0,0 +1,39 @@
+package ro.societateahermes.backendservice.service.serviceImplementation;
+
+import net.sf.saxon.functions.ConstantFunction;
+import org.springframework.stereotype.Service;
+import ro.societateahermes.backendservice.entities.Event;
+import ro.societateahermes.backendservice.repository.EventRepositoryInterface;
+import ro.societateahermes.backendservice.service.dto.NotificationSwitchDTO;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+@Service
+public class EventServiceImplementation {
```

Comment:
A service must extend an Interface so that the implementation is decoupled, easily replaceable or used based on context, also, it is needed to mock services for unit tests
- see SOLID principles (I and D)

Reply:
i don't know how to do that and i would like some help